src/LinuxTracing/LostEventVisitor.h

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef LINUX_TRACING_LOST_EVENT_VISITOR_H_
+#define LINUX_TRACING_LOST_EVENT_VISITOR_H_
+
+#include "LinuxTracing/TracerListener.h"
+#include "PerfEvent.h"
+#include "PerfEventVisitor.h"
+#include "capture.pb.h"
+
+namespace orbit_linux_tracing {
+
+class LostEventVisitor : public PerfEventVisitor {
+ public:
+  void SetListener(TracerListener* listener) { listener_ = listener; }
```

Comment:
Nit: Constructor argument instead?

Reply:
Yeah, this is for consistency with the other `PerfEventVisitor`s. It should be changed for all of them. Let me create a bug and do this in a follow-up change since this is big enough.